docs/development.md

```diff
@@ -18,6 +18,28 @@
 
 # Developer's guide
 
+To run Vagrant development environment, the `$SWAN_DEVELOPEMENT_ENVIRONMENT` variable must be set.
+
+```bash
+export SWAN_DEVELOPEMENT_ENVIRONMENT=true
+cd vagrant
+vagrant up
+```
+
+If Vagrant machine is already provisioned through [Quick Start](/README.md#quick-start) procedure, it should be re-provisioned.
+
+```bash
+export SWAN_DEVELOPEMENT_ENVIRONMENT=true
+cd vagrant
+vagrant provision
+vagrant ssh
+```
+
+Also, thou [Quick Start](/README.md#quick-start) procedure proposes `git clone` of repository, for development purposes it is recommended that repository should be put in `$GOPATH/github.com/intelsdi-x/swan`.
```

Comment:
@skonefal - did you mean "although" instead of "thou"? Consider replacing "proposes" with "suggests".